.changeset/blue-roses-give.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-catalog-react': minor
```

Comment:
I think we can keep all of these changes `patch` changes tbh, since they're not adding new APIs or introducing new features.